src/connection/connection_sr.c

```diff
@@ -1071,6 +1075,32 @@ css_common_connect (CSS_CONN_ENTRY * conn, unsigned short *rid,
   return NULL;
 }
 
+/*
+ * css_make_set_proc_register() - make a server proc register.
+ *   return:
+ *   server_name(in):
+ *   server_name_lenth(in):
+ *   proc_register(out):
+ */
+static void
+css_make_set_proc_register (const char *server_name, int server_name_length, SERVER_PROC_REGISTER * proc_register)
```

Comment:
```suggestion
css_set_proc_register (const char *server_name, int server_name_length, SERVER_PROC_REGISTER * proc_register)
```

받아온 인자값을 설정만 하기 때문에 함수 이름에서 `make`는 제거해도 될 것 같네요.

Reply:
함수명 변경하였습니다.